zkevm-circuits/src/evm_circuit/op_execution/byte.rs

```diff
@@ -161,7 +162,7 @@ mod test {
         ($execution_steps:expr, $operations:expr, $result:expr) => {{
             let circuit =
                 TestCircuit::<Base>::new($execution_steps, $operations);
-            let prover = MockProver::<Base>::run(10, &circuit, vec![]).unwrap();
+            let prover = MockProver::<Base>::run(11, &circuit, vec![]).unwrap();
```

Comment:
What's the reason that `1024` is not enough?

Reply:
It seems there were too many rows for the number of unity roots that is computed out of param 10.